app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command, old_file, new_file = command.split()
+    if old_file == new_file or command != "cp":
+        return
```

Comment:
try not to use blank return here, make a backward condition

Reply:
Like this?